src/visualizers/graph.rs

```diff
@@ -151,12 +149,15 @@ pub trait GraphModifiers {
     ///
     /// ```
     /// Graph::new(cx, Data::gain_mult, (-32.0, 8.0), ValueScaling::Decibels)
-    ///     .fill_from_top()
+    ///     .fill_from_max()
     ///     .color(Color::rgba(255, 0, 0, 160))
     ///     .background_color(Color::rgba(255, 0, 0, 60));
     /// ```
-    fn fill_from_top(self) -> Self;
-
+    fn fill_from_max(self) -> Self {
```

Comment:
I decided to rename `fill_from_top` to `fill_from_max` as to make it a bit more abstract - peak meters can also fill from the right if they're oriented horizontally.